### PR TITLE
AVRO-3878: Rename default git branch to be 'main'

### PR DESCRIPTION
**Pull request by iemejia**

### CODE REVIEW

**Review by opwvhk — APPROVED** [review]
Nice: saves me typing a couple of keys :)

**Comment by iemejia** [comment]
Thanks for pointing out the missing instances. I somehow had a filter on my IDE for `.github` and missed all the others 🤦 !
For the php case I am going to go preventive and rename it to `dev-main`.

**Review by RyanSkraba — APPROVED** [review]
LGTM -- just the one commented URL needs to be restored, and go ahead and merge when you're ready!

I'm glad to make this change. My fingers' muscle memory has definitely also switched my "default" branch to main!

File: lang/csharp/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Avro C# [![Build Status](https://travis-ci.org/apache/avro.svg?branch=master)](https://travis-ci.org/apache/avro) [![NuGet Package](https://img.shields.io/nuget/v/Apache.Avro.svg)](https://www.nuget.org/packages/Apache.Avro)
+# Avro C# [![Build Status](https://travis-ci.org/apache/avro.svg?branch=main)](https://travis-ci.org/apache/avro) [![NuGet Package](https://img.shields.io/nuget/v/Apache.Avro.svg)](https://www.nuget.org/packages/Apache.Avro)
```
[the remark this answers is not in the document]
Comment (martin-g) [reply lang/csharp/README.md:1]: Better replace the badge to point to Github Actions CI.
```
[![Test C#](https://github.com/apache/avro/actions/workflows/test-lang-csharp.yml/badge.svg)](https://github.com/apache/avro/actions/workflows/test-lang-csharp.yml)
```

[the remark this answers is not in the document]
Comment (iemejia) [reply lang/csharp/README.md:1]: Done!

**Pull request merged**